com.unity.multiplayer.mlapi/Runtime/Spawning/NetworkSpawnManager.cs

```diff
@@ -607,31 +607,37 @@ internal void ClientCollectSoftSyncSceneObjectSweep(NetworkObject[] networkObjec
             }
         }
 
-        internal void OnDespawnObject(ulong networkObjectId, bool destroyGameObject)
+        internal void OnDespawnObject(NetworkObject networkObject, bool destroyGameObject)
         {
             if (NetworkManager == null)
             {
                 return;
             }
 
+            if (networkObject == null)
+            {
```

Comment:
This should log a warning or throw probably?

Reply:
Agreed, do you have a preference for throwing an exception or just logging it in a similar way below? I think logging keeps it more consistent with the previous behaviour.

Reply:
Agreed, logging is more consistent.